ext/intl/spoofchecker/spoofchecker_main.c

```diff
@@ -102,9 +106,9 @@ PHP_METHOD(Spoofchecker, setAllowedLocales)
 	size_t locales_len;
 	SPOOFCHECKER_METHOD_INIT_VARS;
 
-	if (FAILURE == zend_parse_parameters(ZEND_NUM_ARGS(), "s", &locales, &locales_len)) {
-		RETURN_THROWS();
-	}
+	ZEND_PARSE_PARAMETERS_START(1, 1)
+		Z_PARAM_STRING(locales, locales_len)
+	ZEND_PARSE_PARAMETERS_END();
 
 	SPOOFCHECKER_METHOD_FETCH_OBJECT;
```

Comment:
Nit/question for another PR, shouldn't the bellow warning use the intl error mechanism? Or is there a reason it can't/shouldn't?

Reply:
I think it should be able to use it, it just seems to be that way since a long time.